app/api/routes.py

```diff
@@ -8,17 +8,26 @@
 from app.utils import Paginator, standardize_response, setup_logger
 from dateutil import parser
 from datetime import datetime
+from prometheus_client import Counter, Summary
 import uuid
 
+# Metrics
+c = Counter('my_failures', 'Number of exceptions raised')
```

Comment:
I'd suggest naming these something more self-explanatory...you may have many counters, and it would be helpful to know which one you're incrementing.